securedrop_client/logic.py

```diff
@@ -67,7 +68,8 @@ def call_api(self):
         if self.i_timed_out is False:
             self.call_finished.emit(result_flag)
         else:
-            logger.info("Thread returned from API call, but it had timed out.")
+            logger.info("Thread returned from API call, "
```

Comment:
note this is basically just for informational purposes for devs right now so this one line I no covered